### PR TITLE
Add try-runtime check in release process

### PR DESCRIPTION
**Pull request by Kailai-Wang**
### Context

As topic, available as standalone CI too

### CODE REVIEW

File: .github/workflows/check-runtime-upgrade.yml [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    types: [released]
+
+  workflow_dispatch:
+    inputs:
+      release_tag:
+        description: runtime.wasm release_tag
+        required: false
+
+env:
+  SUBWASM_VERSION: 0.19.1
```
Comment (BillyWooo) [inline .github/workflows/check-runtime-upgrade.yml:19]: the latest version is v0.21.3. Do we consider using that?

Comment (BillyWooo) [inline .github/workflows/check-runtime-upgrade.yml:19]: the latest version is v0.21.3. Do we consider using that? Any dependency about polkadot version?

Comment (Kailai-Wang) [reply .github/workflows/check-runtime-upgrade.yml:19]: Not in this PR

Comment (Kailai-Wang) [reply .github/workflows/check-runtime-upgrade.yml:19]: We'll need to double-check if the new version has breaking change

File: .github/runtime.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      "uri": "wss://rpc.rococo-parachain.litentry.io:443"
+    },
+    {
+      "name": "litmus",
+      "package": "litmus-parachain-runtime",
+      "path": "runtime/litmus",
+      "uri": "wss://rpc.litmus-parachain.litentry.io:443"
+    },
+    {
+        "name": "litentry",
```
Comment (BillyWooo) [inline .github/runtime.json]: "uri" is the key to determinate whether one entry is valid or not?
Besides, small indentation issue here.

File: .github/workflows/coverage.yml
Comment (BillyWooo) [inline .github/workflows/coverage.yml:1]: not fully understand the changes within this file. Are we going to remove whole "coverage" report? If so, why not a new yml file and file name?

Comment (Kailai-Wang) [reply .github/workflows/coverage.yml:1]: Good catch - I used it to test GHA.

I'm going to revert it

File: .github/runtime.json (hunk 2 above)
Comment (Kailai-Wang) [reply .github/runtime.json]: yes

thanks - I'll fix it

**Pull request merged**